tests/integration/create.bats

```diff
@@ -10,6 +10,116 @@ function teardown() {
 	teardown_bundle
 }
 
+# is_allowed_fdtarget checks whether the target of a file descriptor symlink
+# conforms to the allowed whitelist.
+#
+# This whitelist reflects the set of file descriptors that runc legitimately
+# opens during container lifecycle operations (e.g., exec, create, and run).
+# If runc's internal behavior changes (e.g., new FD types are introduced),
+# this function MUST be updated accordingly to avoid false positives.
+#
+# Allowed patterns include:
+#   - /dev/pts/*                                   : terminal devices for stdio
+#   - anon_inode:[...]                             : eventfd, eventpoll, signalfd, etc.
+#   - "/<proc>"                                    : procfs handle cache
+#   - socket:[<digits>]                            : anonymous sockets used for IPC
+#   - pipe:[<digits>]                              : anonymous pipes used for I/O forwarding
+#   - </proc>/1/task/1/fd                          : procfs fd dir handle cache
```

Comment:
I'm not sure we should be leaking this.

Reply:
Ah no, this is actually because of the re-open of the `exec.fifo` -- we open the parent procfs directory (`/proc/thread-self/fd/`) and then directly open the file descriptor using `openat` but because we are stuck waiting this looks like a leaked file descriptor but it will actually be closed immediately afterwards.